configs/config.d/auth.conf

```diff
@@ -1,6 +1,8 @@
 <source>
   type tail
-  format /^(?<message>(?<time>[^ ]*\s*[^ ]* [^ ]*) .*)$/
+  # Sample format
+  # May  3 18:20:45 localhost sshd[585]: Server listening on 0.0.0.0 port 22.
+  format /^(?<message>(?<time>\w+\s*\d+ \d+:\d+:\d+) .*)$/
```

Comment:
Should `time` be a part of `message` or not? I thought it's more like:

```
/^(?<time>\w+\s*\d+ \d+:\d+:\d+)\s(?<message>.*)$/
```

Reply:
This is something I was dwelling on a bit, I'd really like to have a discussion with you and everyone on this. This is how the line was suggested by the team that requested this. It might be a product decision, but should "we" decide which parts of the "message" are important to keep and display, or should we always assume the entire line is the message, and we simply extract structure from it. WDYT?

Reply:
Seems every other `*.conf` file is excluding info that we already extract. IMO we might want to maintain the consistency. Having the same info in two different places seem like some redundancy anyway. But yeah, it might be a product decision. I would imagine it being a separate effort if we do decide to change it though.

Reply:
Keep in mind the branch we're merging into hasn't been merged upstream yet. I would agree with you that we should be consistent across the other *.conf files as well, so we can make a decision together now which way is the best way to approach this. A nice plus in my mind is that by extracting the entire line as the message, we almost have 100% backwards compatibility as far as the UI is concerned.

Reply:
I'll leave this conversation for the wider effort of structured logging. For now, I'll just do what you've mentioned and merge it in to slide it in for the PE fixit.